genesyscloud/resource_genesyscloud_script.go

```diff
@@ -146,6 +166,15 @@ func createScript(ctx context.Context, d *schema.ResourceData, meta interface{})
 		return diag.Errorf("Script %s not found after creation.", scriptName)
 	}
 
+	//Publish the 	script
+	publishScriptBody := &platformclientv2.Publishscriptrequestdata{
+		ScriptId: sdkScripts[0].Id,
+	}
+
+	if _, _, err := scriptsAPI.PostScriptsPublished("0", *publishScriptBody); err != nil {
```

Comment:
@charliecon How come there is not update script object.  Does the API always just replace the object